### DISCUSSION TITLE
Seperate module versioning from avbroot versioning

### DISCUSSION THREAD

**Discussion by pascallj** [chenxiaolong/avbroot#235] [Ideas]
Currently the module versioning follows the avbroot versioning scheme. Because of this, there exists a lot of module versions which are identical except for the version because the modules don't change that often. It's also unclear if you need to update the modules when a new version of avbroot has been released.

Personally I think the versioning schemes of the modules should be separated from the avbroot versions to make it clear if there actually is something changed in the modules itself.

Additionally it would be nice if the module support the Magisk ['Update JSON format'](https://topjohnwu.github.io/Magisk/guides.html#moduleprop), so the modules can be updated from within the Magisk app if needed (files could be hosted on GitHub, so it doesn't require a separate server). This of course requires the versioning schemes to be separated, otherwise it will prompt the user with unnecessary updates.

**Comment by chenxiaolong** [comment]
Yeah, I agree. I'll probably split out [oemunlockonboot](https://github.com/chenxiaolong/avbroot/tree/master/modules/oemunlockonboot) to its own repo and if I can get #225 working, then [clearotacerts](https://github.com/chenxiaolong/avbroot/tree/master/modules/clearotacerts) won't be needed anymore.

**Reply by chenxiaolong** [reply]
Done! #246
